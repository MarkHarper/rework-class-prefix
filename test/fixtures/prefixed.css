.prfx-my-class {
  color: green;
}

.my-class .prfx-with-another-class {
  box-sizing: border-box;
}
