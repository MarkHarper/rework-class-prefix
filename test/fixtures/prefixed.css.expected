.prfx-my-class {
  color: green;
}

.prfx-my-class .prfx-with-another-class {
  box-sizing: border-box;
}
